samples/fabricaConfig-1org-1channel-1chaincode-tls-raft.json

```diff
@@ -1,6 +1,6 @@
 {
-  "$schema": "https://github.com/softwaremill/fabrica/releases/download/0.0.1-alpha/schema.json",
-  "fabricaVersion": "0.0.1-alpha",
+  "$schema": "https://github.com/softwaremill/fabrica/releases/download/0.0.1/schema.json",
+  "fabricaVersion": "0.0.1",
```

Comment:
a tutaj też jest trochę tricky - na razie będziemy musieli chyba zmieniać z łapy przy zmianie wersji

Reply:
dogadane - jest na to oddzielny issue